openmc/statepoint.py

```diff
@@ -483,7 +510,8 @@ def read_results(self):
                     sum_sq = results[1::2]
 
                 # Define a routine to convert 0 to 1
-                nonzero = lambda val: 1 if not val else val
```

Comment:
Only against them in the context of defining a named function. The argument is that using a lambda to define a named function obviates the usefulness of a lambda over a simple def. PEP8 makes this suggestion actually.